src/sagas/networkSettings.js

```diff
@@ -101,18 +107,28 @@ export function* updateNetworkSettings(action) {
     config.setWalletServiceBaseWsUrl(walletServiceWsUrl);
 
     try {
-      network = yield call(getWalletServiceNetwork);
+      // continue if timeout
+      const { response } = yield race({
+        response: call(getWalletServiceNetwork),
+        timeout: delay(WALLET_SERVICE_REQUEST_TIMEOUT),
+      });
+
+      if (response) {
+        network = response;
+      }
     } catch (err) {
+      // NOTE: Keep the console?
+      console.error('error calling the wallet-service', err);
       rollbackConfigUrls(backupUrl);
```

Comment:
After rollback, don't you need to `return`?

What do you show in the screen in case of a network error?

Reply:
> After rollback, don't you need to `return`?

No, because we try again using the fullnode.

What do you show in the screen in case of a network error?

A failure modal like this:
![ds-customform-failure](https://github.com/HathorNetwork/hathor-wallet-mobile/assets/5992210/e44b286a-b01d-4774-9b9e-0c32a3f9eb16)

Reply:
I found a bug trying to generate this screen:
- fix unhandled promise rejection: 19fb9a5f827578c4b4390a6d8051a8ec51acb3b1